app/src/main/res/values/strings.xml

```diff
@@ -11,5 +11,5 @@
     <string name="Submit">Submit</string>
     <string name="Signin" translatable="false">Sign In</string>
     <string name="Login">Login</string>
-    <string name="SharedPreferencesFilename">com.example.android.oscarine.prefs</string>
+    <string name="SharedPreferencesFilename">oscarine_preferences</string>
```

Comment:
remove it from strings.xml

Constants should not be kept here, only user user visible and resource dependent strings

Reply:
Should I hardcode it then?

Reply:
Yes